src/main/java/com/commercetools/sync/sdk2/products/ProductSyncOptionsBuilder.java

```diff
@@ -0,0 +1,92 @@
+package com.commercetools.sync.sdk2.products;
+
+import com.commercetools.api.client.ProjectApiRoot;
+import com.commercetools.api.models.product.ProductDraft;
+import com.commercetools.api.models.product.ProductProjection;
+import com.commercetools.api.models.product.ProductUpdateAction;
+import com.commercetools.sync.sdk2.commons.BaseSyncOptionsBuilder;
+import javax.annotation.Nonnull;
+
+public final class ProductSyncOptionsBuilder
+    extends BaseSyncOptionsBuilder<
+        ProductSyncOptionsBuilder,
+        ProductSyncOptions,
+        ProductProjection,
+        ProductDraft,
+        ProductUpdateAction> {
+  public static final int BATCH_SIZE_DEFAULT = 30;
+  private SyncFilter syncFilter;
+  static final boolean ENSURE_CHANNELS_DEFAULT = false;
+  private boolean ensurePriceChannels = ENSURE_CHANNELS_DEFAULT;
+
+  private ProductSyncOptionsBuilder(final ProjectApiRoot ctpClient) {
+    this.ctpClient = ctpClient;
+  }
+
+  public static ProductSyncOptionsBuilder of(@Nonnull final ProjectApiRoot ctpClient) {
+    return new ProductSyncOptionsBuilder(ctpClient).batchSize(BATCH_SIZE_DEFAULT);
+  }
+
+  /**
+   * Set option that defines {@link com.commercetools.sync.products.SyncFilter} for the sync, which
```

Comment:
The link to SyncFilter is pointing to the wrong file.

Reply:
380a6f5